lib/src/lib/components/form-fields/date-field/date-field.component.html

```diff
@@ -1,20 +1,40 @@
-<mat-form-field [formGroup]="group" *ngIf="!fieldIsHidden" class="lab900-date-field" id="lab900-date-field-{{ schema?.attribute }}">
+<mat-form-field
+  [formGroup]="group"
+  *ngIf="!fieldIsHidden"
+  class="lab900-date-field"
+  id="lab900-date-field-{{ schema?.attribute }}"
```

Comment:
just wondering, should this not also be changed to `fieldAttribute`?

Reply:
Good catch, fixed :)